index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto&display=swap" rel="stylesheet">
+    <title>Luis Valera's Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <button type="button" class="menu-button">
+             <img src="img/Union.png" alt="menu-image">
+        </button>
+    </header>
+
+    <main class="main">
+        <section class="headline">
+            <h1 id="principal-title" class="color-blue"> Hey there.<br>
+                I'm Luis Carlos.<br>
+                <span>I'm a software developer</span>
+            </h1>
+            <p class="color-blue">I can help you build a product , feature or website Look through some of my work and
+                experience!
+                If you like what you see and have a project you need coded,
+                don't hesitate to contact me.</p>
+            <div class="icons-headline">
+                <img src="img/twitter.png" alt="twitter-icon">
+                <img src="img/linkedin.png" alt="linkedin-icon">
+                <img src="img/medium.png" alt="medium-icon">
+                <img src="img/github.png" alt="github-icon">
+                <img src="img/angellist.png" alt="angellist-icon">
+            </div>
```

Comment:
- Kindly, Use a `ul` tag to group the list of icons, and `li` per icon; each icon has a link (`a` tag), this conversion used Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href` :+1: .